code/datums/mood_events/generic_negative_events.dm

```diff
@@ -245,3 +245,8 @@
 	description = "<span class='boldwarning'>I'm not safe! I can't trust anybody!</span>\n"
 	mood_change = -6
 	timeout = 30 SECONDS
+
+/datum/mood_event/horo_bad
+	description = "<span class='warning'>I'm doomed! Woe be upon me!"
```

Comment:
No `</span>`
Seriously, how have you not picked up this habit yet

Reply:
idk!